spoj_practice/spoj2/spoj2.py

```diff
@@ -7,43 +8,29 @@
 # N < 1,000,000,000
 # Find all primes between a given n, m
 
-
 # handle stdin/stdout and unit testing
 
-
-
 # returns a list of all the primes in between
 def find_primes_between (n, m):
     # divide by all odd numbers, counter it up
     # divide by all primes before it
     # divide by all other primes between n, m
     # make sure we start with an odd number
-    
+
     # primes to check against
     # 1 is not a prime number, the loop excludes even numbers thus 2 does not need to be added
-    primes = [3]
-    primes_between = []
-    
-    # unique cases
-    if m < 2:
-        primes_between.append(2)
-    if m <= 3:
-        primes_between.append(3)
-    
+    primes = [2, 3]
```

Comment:
Probably easiest for me to explain over skype or IRL. Can you record these outstanding questions in a document somewhere so we can review next time?